letsencrypt/build.yaml

```
@@ -8,6 +8,7 @@ codenotary:
   signer: notary@home-assistant.io
   base_image: notary@home-assistant.io
 args:
+  CERTBOT_DNS_AZURE_VERSION: 1.4.0
```

```suggestion
  CERTBOT_DNS_AZURE_VERSION: 1.5.0
```
I realize this is just because this has been sitting open for a while but since 1.5.0 has now come out we should start on the latest stable version unless it causes some issue.

---

Done.